modules/claims_api/spec/sidekiq/v2/disability_compensation_benefits_documents_uploader_spec.rb

```diff
@@ -116,4 +116,17 @@
       end
     end
   end
+
+  describe 'when an errored job has a time limitation' do
+    it 'logs to the ClaimsApi Logger' do
+      error_msg = 'An error occurred from the BD Uploader Job'
+      msg = { 'args' => [claim.id],
+              'class' => subject,
+              'error_message' => error_msg }
+
```

Comment:
@stiehlrod  Is the logging actually being tested somewhere from these tests?  I see the time limit being tested but there is no expectation set for the log out, which I believe would come from inside the exhausted block in this event?  I also believe the exhausted block is already being tested for these anyways so the logging check is probably not even needed for any of these tests (?)  Let me know if I am missing something there, but the `msg` variable and the `error_msg` can probably just be removed from this since I'm not sure they need to be tested again and it does not look like they are being used in these tests currently.